plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py

```diff
@@ -256,7 +256,16 @@ def _onPrintUploadCompleted(self, response: CloudPrintResponse) -> None:
         """
         self._uploaded_print_job = self._pre_upload_print_job
         self._progress.hide()
-        PrintJobUploadSuccessMessage().show()
+        message = PrintJobUploadSuccessMessage()
+        message.addAction("monitor print",
+                          name=I18N_CATALOG.i18nc("@action:button", "Monitor Print"),
+                          icon="",
+                          description=I18N_CATALOG.i18nc("@action:tooltip", "Track the print in Ultimaker Digital Factory"),
+                          button_align=message.ActionButtonAlignment.ALIGN_RIGHT)
+        df_url = "https://digitalfactory.ultimaker.com/app/jobs/%s?utm_source=cura&utm_medium=software&utm_campaign=monitor-button" % self._cluster.cluster_id
```

Comment:
Why not use an f-string here?
```suggestion
        df_url = f"https://digitalfactory.ultimaker.com/app/jobs/{self._cluster.cluster_id}?utm_source=cura&utm_medium=software&utm_campaign=monitor-button"
```

Reply:
Because i'm still not used to them yet :P